website/docs/advanced-concepts/sharing-data-across-pages.md

```diff
@@ -6,41 +6,83 @@ description: >-
 ---
 
 # Share Data Across Pages
-You can share data between pages by utilizing query parameters or Appsmith's store value, which is a global state object. This page explains how to share data between different pages in your Appsmith applications.
 
-## Use query parameters
-Use the query parameter to share data between pages across apps. To pass data between the source and the target page using query parameters, use the [navigateTo()](/reference/appsmith-framework/widget-actions/navigate-to) function.
+This guide shows how to share data across different pages in Appsmith using the `navigateTo()` or the `storeValue()` function.
 
-Example:
+## Using navigateTo() Function
 
-In the following example, `page_name` is the name of the target page. 
-```jsx
-{{navigateTo('page_name', { userId: 547916 })}}
+The [navigateTo()](/reference/appsmith-framework/widget-actions/navigate-to) function allows you to navigate between internal app pages or external URLs while passing data as query parameters, such as user inputs or query data.
+
+<dd>
+
+<ZoomImage src="/img/sharedata-page.png" alt="Navigate to action" caption="Navigate to action" />
+
+</dd>
+
+
+1. Select **Navigate to** from the action selector.
```

Comment:
I think we shouldn't use action selector, users may not understand this terminology. Also, let's use the same example that's shown in the screenshot -- where the navigateTo is bound to an onclick event. Let's elaborate on that example.